crates/flipperzero/src/furi/time.rs

```diff
@@ -464,9 +470,36 @@ impl<'a> Sum<&'a Duration> for Duration {
     }
 }
 
+#[derive(Clone, Copy, Debug, PartialEq, PartialOrd, uDebug)]
+pub struct TryFromDurationError;
+
+impl core::error::Error for TryFromDurationError {}
+
+impl core::fmt::Display for TryFromDurationError {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
```

Comment:
Nit: I suggest importing `core::fmt` in scope

Reply:
Fixed. I did the same for the other `core::` uses.